mantidproject/mantid
# Add single function to read embedded parameters associated with nexus file

Fixes issue #12940 

To test
Review code
Check that unit tests are adequate
Check that Mantid runs as normal when loading nexus files that may use parameters

Release notes: This issue is part of a greater issue #12255 so does not need its own release notes.
